docs/cli-usage.md

```diff
@@ -0,0 +1,270 @@
+## Usage 
+
+tobs CLI installation
+
+`curl --proto '=https' --tlsv1.2 -sSLf  https://tsdb.co/install-tobs-sh |sh`
+
+Available Commands:
+
+```
+$ tobs
+The Observability Stack is a tool that uses TimescaleDB as a 
+compressed long-term store for time series metrics from Prometheus. This
+application is a CLI tool that allows users to quickly access the different
+components of Observability.
+
+Usage:
+  tobs [command]
+
+Available Commands:
+  grafana      Subcommand for Grafana operations
+  helm         Subcommand for Helm operations
+  help         Help about any command
+  install      Alias for helm install
+  metrics      Subcommand for metrics operations
+  port-forward Port-forwards TimescaleDB, Grafana, and Prometheus to localhost
+  prometheus   Subcommand for Prometheus operations
+  promlens     Subcommand for Grafana operations
+  timescaledb  Subcommand for TimescaleDB operations
+  uninstall    Alias for helm uninstall
+
+Flags:
+      --config string      config file (default is $HOME/.tobs.yaml)
+  -h, --help               help for tobs
+  -n, --name string        Helm release name (default "tobs")
+      --namespace string   Kubernetes namespace (default "default")
+
+Use "tobs [command] --help" for more information about a command.
+```
+
+### Deploying monitoring stack
+
+`tobs install`
+
+This will successfully deploy all the above listed components to your k8s cluster.
+
+You can use `--name` flag to set helm release name default name is `tobs`, `--namespace` flag to deploy observability stack in namespace of your choice default is set to `default` namespace, `--chart-reference` flag to use helm chart of your choice default is set to `timescale/tobs`, `--filename` to use values.yml file of your choice with helm chart.
+
```

Comment:
Maybe add a sentance here that tells people that the values.yml file is what they would use with manual helm deployes and where and how they can generate the default file

Reply:
The default file exists in `chart/values.yaml` right? I am not sure how can one generate values.yaml file?